lib/core/utils.js

```diff
@@ -361,11 +361,20 @@ RESUtils.thingsContainer = function(body) {
 
 	return container;
 };
-RESUtils.things = function(container) {
+RESUtils.$things = function(container) {
 	container = container || RESUtils.thingsContainer();
 	return $(container).find(RESUtils.thing.prototype.thingSelector);
 };
+RESUtils.things = function(container) {
```

Comment:
It's only used in NER so I didn't have to rewrite .thing or a lot of code that expects a regular jQuery element set.